doc/sources/oneapi_gpu.rst

```diff
@@ -0,0 +1,100 @@
+   <https://intelpython.github.io/dpctl/latest/docfiles/dpctl.tensor_api.html#dpctl.tensor.usm_ndarray>`_
+   to the algorithm. The computation will run on the device where input data is
+   located, and result will be returned as :code:`usm_ndarray` on the same
+   device.
+
+   .. note::
+     All the data passed to the algorithm must reside on the same device.
+
+   .. warning::
+     Only base methods like :code:`fit`, :code:`predict`, and :code:`transform`
+     can consume :code:`usm_ndarray`. Please pay attension that only the
+     algorithms in Intel(R) Extension for Scikit-learn* provide support of
+     :code:`usm_ndarray`. Standard algorithms in stock version of Scikit-learn*
+     do not include this feature.
+2. Use global configurations of Intel(R) Extension for Scikit-learn*:
+     1. Option :code:`target_offload` can be used to set the device primarily
+        used to perform computations. Accepted values are :code:`str` and
+        :code:`dpctl.SyclQueue`. If string, expected to be "auto" (the execution
+        context is deduced from input data location), or SYCL* filter selector
+        string. Default value is "auto"
+     2. Option :code:`allow_fallback_to_host`
+        is a boolean flag, that, if set, allows to fallback computation to host
+        device in case when particular estimator does not support the selected
+        one. Default value is :code:`False`.
+
+These options can be set using :code:`sklearnex.set_config()` function or
+:code:`sklearnex.config_context`. Current values of these options can be
+obtained by the call of :code:`sklearnex.get_config()`.
+
+.. note::
+     Functions :code:`set_config`, :code:`get_config` and :code:`config_context`
+     are always patched after :code:`sklearnex.patch_sklearn()` call.
+
+Example how to patch your code with Intel CPU/GPU optimizations:
+
+.. code-block:: python
+
+   from sklearnex import patch_sklearn, config_context
+   patch_sklearn()
+
+   from sklearn.cluster import DBSCAN
+
+   X = np.array([[1., 2.], [2., 2.], [2., 3.],
+               [8., 7.], [8., 8.], [25., 80.]], dtype=np.float32)
+   with config_context(target_offload="gpu:0"):
+      clustering = DBSCAN(eps=3, min_samples=2).fit(X)
+
+.. warning::
+     For compatibility reasons, algorithms in Intel(R) Extension for
+     Scikit-learn* can be offloaded to the device with use of
+     :code:`daal4py.oneapi.sycl_context`. It is recommended to use the ways
+     described above for device offloading instead of using :code:`sycl_context`.
+
+
+For execution on GPU, DPC++ compiler runtime and driver are required. Refer to `DPC++ system
+requirements <https://software.intel.com/content/www/us/en/develop/articles/intel-oneapi-dpcpp-system-requirements.html>`_ for details.
+
+DPC++ compiler runtime can be installed either from PyPI or Anaconda:
+
+- Install from PyPI::
+
+     pip install dpcpp-cpp-rt
+
+- Install from Anaconda::
+
+     conda install dpcpp_cpp_rt -c intel
```

Comment:
lines 89-100 should be placed at the beginning of this topic and clearly marked as "requirements" or "prerequisites"